### DISCUSSION TITLE
Get Headers is not defined when using createRemoteJWKSet

### DISCUSSION THREAD

**Discussion by mtubul** [panva/jose#638] [Q&A]
I am using jose version 5.2.1, 
this is my code
```
 const remoteJwks = jose.createRemoteJWKSet<any>(new URL(this.remoteJWKURL));
      const res = await jose.jwtVerify(jwt, remoteJwks, options);
      return res;
```

the above code throw the following error 
```
ReferenceError: Headers is not defined
```

as for now i found it throw in line https://github.com/panva/jose/blob/main/src/jwks/remote.ts#L140
any idea how can i solve it?
there is any dependency i missed?

**Comment by big-kahuna-burger** [comment]
What's your output of `node -v` ?

Migrate to version 20.

**Reply by mtubul** [reply]
@big-kahuna-burger thank you, it is working. there is any announcement that version 5.2.1 depends on node version 20?

**Reply by panva** [reply]
https://github.com/panva/jose/releases/tag/v5.0.0#:~:text=End%2DOf%2DLife,-versions%20of%20Node

**Reply by mtubul** [reply]
thanks
